admin-dev/themes/default/template/controllers/search/helpers/view/view.tpl

```diff
@@ -151,16 +151,24 @@ $(function() {
 
 {/if}
 <div class="row">
-    <div class="col-lg-6">
+    <div class="col-lg-{if $alternativeSearchPanels|@count == 0}6{else}4{/if}">
         <div class="panel">
             <h3>{l s='Search doc.prestashop.com' d='Admin.Navigation.Search'}</h3>
```

Comment:
@Julievrz is is possible to change this string ?

Reply:
Yes we can, we should! :-)